ocp-gitlab-runner-template.yaml

```diff
@@ -210,7 +217,7 @@ objects:
             - name: RUNNER_NAME
               value: ${NAME}
             - name: CI_SERVER_URL
-              value: ${CI_SERVER_URL}
+              value: https://${CI_SERVER_URL}
```

Comment:
Adding this helps me re-use the `CI_SERVER_URL` below in secret object. Else I would have needed another parameter afaiu.